### PR TITLE
Project: Building with Responsive Design 1st submition for review

### PR DESCRIPTION
**Pull request by jubaan**
Our approach was wider screens first and last mobile. We manage to set up the layout and the changes required for each one of the breakpoints. We found that TNW only uses 2 breakpoints smaller<768-1024<wider. We didn't put that much attention on the details, because the main goal was the layout and the use of media queries.

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!

### Your project looks good!  :+1: 
#### But there are a few things that need to be corrected:

#### Page structure

##### Navigation bar in Small version changes:
* Navigation and sub-navigation row should change to a hamburger (bars) icon at the rightmost part (next to search icon) ([link to page](https://thenextweb.com/))

#### Original website in small version contains a hamburger icon
original

#### Your clone in small version is without hamburger icon, kindly recommended to add it.
nav


#### CSS styles should use media queries for 3 breakpoints (screenshot from Pathwright)
screen

* As I can see in your project just for 2 breakpoints @media only screen and (max-width: 768px) and 
@media only screen and (max-width: 1024px). Kindly recommended to add else one.


#### Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)


* Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+  height: 239.7px;
+}
+
+.article-big .cover-article-info {
+  margin: 20px;
+}
+
+.article-big .article-section {
+  color: white;
+  font-size: 0.87 em;
```
Comment (Anna-Myzukina) [inline css/style.css]: Line number 290 : invalid property value 
Kindly recommended to delete space (write 0.87em instead 0.87 em)

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,799 @@
...
+  justify-content: flex-end;
+}
+
+.article-small .cover-article-info {
+  margin: 20px;
+}
+
+.article-small .article-section {
+  color: white;
+  font-size: 0.87 em;
```
Comment (Anna-Myzukina) [inline css/style.css]: Line number 321 : invalid property value 
Kindly recommended to delete space (write 0.87em instead 0.87 em)

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,799 @@
...
+.section-details {
+  flex: 0 1 33.333%;
+  max-width: 100%;
+  padding: 8px 20px;
+  margin: 0;
+}
+
+.section-details .first-article {
+  background: url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/12/JBL-Reflect-Eternal-hed-368x276.jpg);
+  background-size: 100% 100% cover;
```
Comment (Anna-Myzukina) [inline css/style.css]: Line number 455 here is invalid property value, kindly recommended to fix it.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,799 @@
...
+    display: none;
+  }
+
+  .article-big .cover-article-info {
+    margin: 20px;
+  }
+
+  .article-big .article-section {
+    color: white;
+    font-size: 0.87 em;
```
Comment (Anna-Myzukina) [inline css/style.css]: Line number 731 : invalid property value
Kindly recommended to delete space (write 0.87em instead 0.87 em)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1059 @@
...
+          </p>
+        </div>
+      </article>
+    </section>
+
+    <section class="latest-news">
+      <div class="latest-news-headline">
+        <h2>
+          <a href="javascript:void(0)">
+            Latest news >
```
Comment (Anna-Myzukina) [inline index.html]: Line number 120: Special characters must be escaped : [ > ]. Kindly recommended to delete symbol > after "Latest news"

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project needs changes

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Nice job @jubaan and @certilremy! :rocket: 
There are a few things that need your attention before this project can be approved :smile: 

#### Horizontal Scroll
Your CSS is a little buggy, it has some horizontal scrolling going on.

#### Wrongly Sized Elements
A lot of the elements on the page are wrongly sized, please take a close look at the original size and adjust each element appropriately.

#### Missing Elements
You have a few things wrong with your navbar, elements are missing and it isn't the most responsive. Please look into this.

### Summary
You have done a nice job getting this far! Please take a closer look at your site's elements and responsiveness. Also, take a look at each element's design and try to adjust each appropriately! When all this is done, please request a re-review :+1:

Please reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin) if you think I missed something or you need some help!

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work @jubaan and @certilremy!
You've implemented most of the changes beautifully, and there is just one more change you need to make. The section just beyond the latest funding section needs a bit of work :+1:, please look closely at the original and make the appropriate changes.

When you're done with that, please request a re-review!

**Comment by certilremy** [comment]
Hello Tolulope Oluwadamilare Olusakin Thanks for your feedback . Can you please send us a capture or tell us briefly the chages that we need to do ?

**Comment by certilremy** [comment]
> # Great work @jubaan and @certilremy!
> You've implemented most of the changes beautifully, and there is just one more change you need to make. The section just beyond the latest funding section needs a bit of work 👍, please look closely at the original and make the appropriate changes.
> 
> When you're done with that, please request a re-review!

Hello Tolulope Oluwadamilare Olusakin Thanks for your feedback . Can you please send us a capture or tell us briefly the chages that we need to do ?

**Comment by nikasulo** [comment]
Thanks for reaching out @certilremy ! I will send a screenshot on slack.

**Comment by certilremy** [comment]
> Thanks for reaching out @certilremy ! I will send a screenshot on slack.

Hello . Dont forget us , we are waiting.

**Comment by shshamim63** [comment]
## Status :tada:  [!! Approved !!] :tada: 
___
Hi @certilremy  and @jubaan :wave: .
Awesome work. No lint error, no HTML markup validation error. Code looks neat and clean. Readme is descriptive enough to guide me about this project. PR contain all the specific information about tasks that are covered in this PR. You have followed all the task provided by @Anna-Myzukina. Nothing is blocking me to declare this project as complete.
___
##### This :+1: is from me. You can merge it. Don't forget to fill the project completion form.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](__Shakhawat Hossain__)

**Pull request merged**